integration/screenboards_test.go

```diff
@@ -93,8 +93,8 @@ func TestScreenboardGet(t *testing.T) {
 func getTestScreenboard() *datadog.Screenboard {
 	return &datadog.Screenboard{
 		Title:   datadog.String("___Test-Board___"),
-		Height:  datadog.Int(600),
-		Width:   datadog.Int(800),
+		Height:  datadog.JsonNumber("600"),
+		Width:   datadog.JsonNumber("800"),
```

Comment:
Similar to #194, this code will fail downstream during marshalling, right before sending the payload to the client. Try setting `Width:   datadog.JsonNumber("100%"),` here to expose the issue.

Reply:
Thanks for review.
Fixed it.